ime/app/src/main/res/values/arrays.xml

```diff
@@ -59,6 +59,18 @@
         <item>upper</item>
     </string-array>
 
+    <string-array name="settings_key_hint_size_names">
+        <item>@string/settings_key_hint_size_small</item>
```

Comment:
Can you also add 'none'? Which will force it to invisible.

Reply:
Yeah, i can do that. But at the beginning I thought that if the user didn't want it, he could change the theme.